Gemfile

```diff
@@ -9,12 +9,14 @@ gem 'haml'
 gem 'sass'
 
 gem 'sqlite3'
+gem 'rqrcode'
 
 group :development do
   gem 'byebug'
 end
 
 group :test do
+  gem 'rspec'
```

Comment:
assert page.has_css? "img[src*='/admin/items/#{@items[7].id}.png']"

I was unable to easily test for an image on the page.